src/Resources/views/CRUD/base_history.html.twig

```diff
@@ -52,40 +51,8 @@ file that was distributed with this source code.
                 </div>
             </div>
         </div>
-        <div id="revision-detail" class="col-md-7 revision-detail">
+        <div id="revision-detail" class="col-md-7 revision-detail" {{ stimulus_target('sonata-revision', 'preview') }}>
 
         </div>
     </div>
-
-    <script>
-        jQuery(document).ready(function() {
-
-            jQuery('a.revision-link, a.revision-compare-link').bind('click', function(event) {
-                event.stopPropagation();
-                event.preventDefault();
-
-                action = jQuery(this).hasClass('revision-link')
-                    ? 'show'
-                    : 'compare';
-
-                jQuery('#revision-detail').html('');
-
-                if(action == 'show'){
-                    jQuery('table#revisions tbody tr').removeClass('current');
-                    jQuery(this).parent('').removeClass('current');
```

Comment:
IMHO its dead code, because class `current` not sets in backend and not updates in js

Reply:
I wonder if this class is not added by some third parties.
This feel like an unwanted behavior when clicking on a link.

Reply:
Looking at commit [baa6cd0](https://github.com/sonata-project/SonataAdminBundle/commit/baa6cd0ee9bc9fef43d90eef8dcd621b8817342e#diff-a54cfc7be22b1f5b64bc51986f750218107ef545d38ec29d1a09653dae136144) this was added initially